src/test/java/shop/gitit/member/service/UpdateNickNameServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package shop.gitit.member.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import shop.gitit.member.domain.Member;
+import shop.gitit.member.domain.support.member.MemberFixture;
+import shop.gitit.member.repository.MemberRepository;
+import shop.gitit.member.service.dto.request.UpdateMemberNickNameReqDto;
+import shop.gitit.member.service.dto.response.UpdateMemberNickNameResDto;
+import shop.gitit.member.service.usecase.UpdateNickNameUsecase;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {UpdateNickNameService.class})
+class UpdateNickNameServiceTest {
+    @Autowired private UpdateNickNameUsecase updateNickNameUsecase;
+
+    @MockBean private MemberRepository memberRepository;
+
+    @DisplayName("updateNickName 메서드는 기릿으로 닉네임을 변경한다.")
```

Comment:
```suggestion
class UpdateNickNameServiceTest {

    @Autowired private UpdateNickNameUsecase updateNickNameUsecase;
    @MockBean private MemberRepository memberRepository;
    
    @DisplayName("updateNickName 메서드는 기릿으로 닉네임을 변경한다.")
```
인터페이스와 클래스명, 메서드 사이에 공백은 컨벤션으로

Reply:
오케이